source/OutfitterPanel.cpp

```diff
@@ -112,6 +116,13 @@ void OutfitterPanel::Step()
 		if(!DoHelp("outfitter") && !DoHelp("outfitter 2") && !DoHelp("outfitter 3"))
 			// All help messages have now been displayed.
 			checkedHelp = true;
+
+	for(const auto &outfit : outfitter)
```

Comment:
We're iterating through every outfit in the outfitter on every frame, not ideal. You could move this to the constructor.

Reply:
will take a look